static/src/javascripts/projects/common/modules/commercial/contributions-utilities.js

```diff
@@ -102,11 +98,13 @@ const getTestimonialBlock = (
 const defaultPageCheck = (page: Object): boolean =>
     page.contentType === 'Article' && !page.isMinuteArticle;
 
-const defaultCanEpicBeDisplayed = (test: ContributionsABTest): boolean => {
-    const canReasonablyAskForMoney =
-        test.showToContributorsAndSupporters ||
-        commercialFeatures.canReasonablyAskForMoney;
+const shouldShowReaderRevenue = (
+    showToContributorsAndSupporters: ?boolean = false
```

Comment:
I'm surprised you need to use a maybe type here -- may be linked to [a bug in babel](https://github.com/babel/babel/issues/2493).